### PR TITLE
Update Firestore changelog for M68

### PR DESCRIPTION
**Pull request by morganchen12**

### CODE REVIEW

File: Firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# v1.11.3
+- [changed] Changed the default log level to `notice` instead of `warning` (#5196).
```
Comment (wilhuff) [inline Firestore/CHANGELOG.md:4]: This only applies to non-Apple-platform users so it doesn't belong in this changelog.

File: Firestore/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+- [added] The Firestore iOS SDK can now be built via CMake (#5052).
```
Comment (wilhuff) [inline Firestore/CHANGELOG.md:5]: This isn't supported. We have a CMake build to make it possible for us to combine this code with the C++ SDK, but that's an implementation detail of the C++ SDK, not a supported usage path for Apple platform users.

Comment (morganchen12) [reply Firestore/CHANGELOG.md:5]: Removed.

File: Firestore/CHANGELOG.md (hunk 1 above)
Comment (morganchen12) [reply Firestore/CHANGELOG.md:4]: Removed.

**Pull request merged**